Add requirement python version 3.9 to setup.py

DunklesArchipel opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check. Remove checks that are not relevant and let us know if you need help with any of these.
-->
Checklist
* [x] Added an entry in `doc/news/`. <!-- Copy the TEMPLATE.rst to mybranch.rst, fill in the relevant sections, delete the others. -->
* [x] Added a test for this change. -> not required
* [x] Adapted the documentation for this change. -> not required

<!--
Please add any other relevant info below:
-->

Addressed #36

DunklesArchipel committed:
Update python version

DunklesArchipel commented on setup.py:
```diff
@@ -38,5 +38,6 @@
       "pandas>=1,<2",
       "plotly>=5,<6",
       "pybtex>=0.24,<0.25",
+      "python>=3.9,<4"
```
```suggestion
      "python>=3.9,<4",
```

DunklesArchipel committed:
Update setup.py

DunklesArchipel committed:
fix syntax

DunklesArchipel committed:
Merge remote-tracking branch 'DunklesArchipel/setuppython' into setuppython

DunklesArchipel committed:
fix python version

DunklesArchipel committed:
update news

saraedum commented on doc/news/setuppython.rst:
```diff
@@ -0,0 +1,3 @@
+**Added:**
```
```suggestion
**Fixed:**
```

saraedum committed:
Update doc/news/setuppython.rst

While this "adds" something literally. It does not "add" something to this library in the way that people expect (features to be added.)
